### PR TITLE
Update terraform-provider-libvirt version to v0.6.0.

### LINKED ISSUE #2499: Updating Gopkg.toml for terraform-provider-libvirt version 0.6.0
s390 platform by libvirt needs the support of adding coreos ignition through guestfs.
However, terraform-provider-libvirt v0.5.2 doesn't support this but v0.6.0 could.
So terraform-provider-libvirt version should be modified in Gopkg.toml to update it's version.

### PR DESCRIPTION
**Pull request by xkwangcn**
Update terraform-provider-libvirt version to v0.6.0.

The terraform-provider-libvirt v0.6.0 contains the support of guestfs for s390 platform.
So this PR is to update erraform-provider-libvirt version.

Fixes #2499.

### CODE REVIEW

**Comment by stbenjam** [comment]
Kicking off metal3 CI as this affects us, too.

/label platform/baremetal

**Comment by metal3ci** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/1224/

**Comment by xkwangcn** [comment]
hi, could you help to review it? if any problem, I will modify it in time to merge the PR earlier. @zeenix @abhinavdahiya

**Comment by gyliu513** [comment]
I think the best practise is that we should make the generated code in a separate PR based on guidance here https://github.com/openshift/installer/blob/master/docs/dev/dependencies.md#go

image

File: pkg/terraform/exec/plugins/Gopkg.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ ignored = [
 
 [[constraint]]
   name = "github.com/dmacvicar/terraform-provider-libvirt"
-  revision = "c0e46b59df8718cdd905b1a3fb9738b0d4905143"
+  version = "=0.6.0"
```
Comment (praveenkumar) [inline pkg/terraform/exec/plugins/Gopkg.toml]: LGTM

**Pull request merged**